src/21-pubsub/index.md

```diff
@@ -0,0 +1,116 @@
+---
+# this is the final blog post's id (used in the directory)
+# this id is part of the url, and should only contain:
+#   letters, numbers, dashes.
+id: 25-pubsub
+
+breadcrumbs:
+  - {name: "Take a look at pubsub on IPFS", link: "./" }
+
+# the date here should be set to the final publication date,
+# on the day it is published.
+date: 2017-05-16
+
+# this is the Title
+title: Take a look at pubsub on IPFS
+
+# this is the name of the main author(s)
+author: Jeromy Johnson
+
+# technical details required for the software, don't change these.
+baseurl: ..
+template: tmpl/layouts/post.html
+collection: posts
+---
+
+To make IPFS fast for datacenters, local area networks, and large p2p
```

Comment:
These aren't the main value propositions. The main value proposition of pubsub is that it's one way of bridging the gap between immutable and mutable. It allows people to create fast, responsive, distributed applications that operate in real-world contexts where data are constantly being updated.